ckan/templates-bs2/group/members.html

```diff
@@ -25,10 +25,10 @@ <h3 class="page-heading">{{ _('{0} members'.format(c.members|length)) }}</h3>
         <td>{{ role }}</td>
         <td>
           <div class="btn-group pull-right">
-            <a class="btn btn-small" href="{% url_for controller='group', action='member_new', id=c.group_dict.id, user=user_id %}" title="{{ _('Edit') }}">
+            <a class="btn btn-small" href="{% url_for controller='group', action='member_delete', id=c.group_dict.id, user=user_id %}" title="{{ _('Edit') }}">
```

Comment:
@amercader action should me member_new

Reply:
Thanks @tino097, can you review and merge https://github.com/ckan/ckan/pull/4046? cheers